aspnetcore/hosting/url-rewriting.md

```diff
@@ -1,18 +1,303 @@
 ---
-title: 🔧 URL rewriting middleware | Microsoft Docs
+title: URL Rewriting | Microsoft Docs
 author: guardrex
-description: 
-keywords: ASP.NET Core,
+description: An introduction to URL rewriting and redirecting with instructions on how to use URL Rewriting Middleware in ASP.NET Core applications.
```

Comment:
"URL Rewriting Middleware" is a proper, singular noun and refers to .NET-specific technology. I think the definite article is optional here, and it's probably more conversational to not have it. I would speak this sentence without it. We can take a tie-breaker with Chris, Tom, and Rick. :smile: